fast/extras/0-cicd-github/main.tf

```diff
@@ -143,8 +144,8 @@ resource "github_repository_file" "default" {
     endswith(each.value.name, ".tf") && local.modules_repo != null
     ? replace(
       file(each.value.file),
-      "/source\\s*=\\s*\"../../../modules/([^/\"]+)\"/",
-      "source = \"git@github.com:${local.modules_repo}.git//$1${local.modules_ref}\"" # "
+      "/source(\\s*)=\\s*\"../../../(modules/[^/\"]+)\"/",
+      "source$1= \"git@github.com:${local.modules_repo}.git//$2${local.modules_ref}\"" # "
```

Comment:
Anton, what we target is a modules-only repository, where the `modules/` prefix is not used. Can you revert this change?

Reply:
Of course.

Reply:
Thanks. :) Thinking about it, we might want to introduce an optional "module prefix" in the variables, to account for different usecases. Feel free to do it, or ignore and we can do it later in a subsequent PR. Thanks a lot for this and just let me know when you feel like it can be merged.

Reply:
@ludoo , could you please review the 'module_prefix' attribute I created in 'modules_config'? It's intended to handle use cases for non-root modules.

Reply:
Thanks! Looks great. :)